rethinkdb/ast.py

```diff
@@ -92,7 +92,7 @@ def expr(val, nesting_depth=20):
 
     if isinstance(val, RqlQuery):
         return val
-    elif isinstance(val, collections.Callable):
+    elif isinstance(val, collections.abc.Callable):
```

Comment:
change also needed for collections.Mapping

Reply:
This was fixed with #253.